openshift/clowder/clowd-app.yaml

```diff
@@ -120,6 +120,10 @@ objects:
             value: ${ENABLE_SIGNING}
           - name: GNUPGHOME
             value: ${GNUPGHOME}
+          - name: PULP_GALAXY_SIGNATURE_UPLOAD_ENABLED
+            value: ${GALAXY_SIGNATURE_UPLOAD_ENABLED}
+          - name: PULP_GALAXY_REQUIRE_SIGNATURE_FOR_APPROVAL
```

Comment:
@rochacbruno with this setting enabled in CRC, we may need updates to the integration tests that approve collections in insights mode, since they will fail if the collection is not signed

Reply:
In the case of ephemeral, should I set GALAXY_AUTO_SIGN_COLLECTIONS?

Reply:
yeah, that might be all we need